pkg/antctl/handlers/agentinfo.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/monitor"
+)
+
+var _ Factory = new(AgentInfo)
+
+// ComponentAgentInfoResponse describes the internal response struct of agent-info command.
+// It only contains agent's node subnet and connection status of controller, OVSDB and openflow.
```

Comment:
I think you can remove this second line of comment: what is included in the response is clear from the structure definition and we should not have to update the struct-level comment every time we add some extra information to the response. However, you may want to add some comments to the struct members when needed (e.g.: "Subnet is the subnet range from the Pod CIDR allocated to the Node" or something like this).

Reply:
I agree. Updated.